src/components/card/card.njk

```diff
@@ -18,9 +18,13 @@
           {{ params.aboveContent.html | safe }}
         </div>
       {%- endif -%}
-      <a href="{{ params.href }}" class="nhsapp-card__link nhsuk-link--no-visited-state">
-        {{ params.title }}
-      </a>
+      {%- if params.readOnly -%}
+        <h3 class="nhsapp-card__title">{{ params.title }}</h3>
```

Comment:
@davidhunter08 should this heading support more than just `h3`?

Reply:
I did a quick syntax checker and it came up with a warning when using a `h3`:

> h3 may be used for formatting

<img width="352" alt="Screenshot 2024-09-11 at 12 36 01" src="https://github.com/user-attachments/assets/39bd0306-d38c-4961-8c1e-d574d34fee5a">

We [don't use headings in card links](https://github.com/nhsuk/nhsuk-service-manual-community-backlog/issues/249#issuecomment-2011772269) so having it as a `<p>` should be ok.